project_task_default_stage/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     'name': 'Project Task Default Stage',
     'summary': 'Recovery default task stage projects from v8',
-    'version': '10.0.1.0.0',
+    'version': '10.0.2.0.0',
```

Comment:
Yeah, but I think you have read incorrectly the guide. The change I propose is for changes that requires a module update.